CONTRIBUTING.md

```diff
@@ -0,0 +1,184 @@
+# Contribution Guidelines
+
+Thank you for expressing interest in contributing to the development and
+maintenance of Crust firmware. The success of the Crust firmware project is
+made possible by community support of source code and documentation. This
+document contains guidelines and procedures for contributing to the Crust
+firmware project.
+
+## Code of Conduct
+
+This project is governed by the [Contributor Covenant Code of Conduct][coc].
+By contributing to Crust firmware, you are entering an agreement to uphold all
+guidelines and responsibilities outlined within. Please carefully read and
+review the agreement before contributing.
+
+## Issue Tracking
+
+All bugs and features are tracked through the Crust firmware GitHub [issues
+page][issues]. When filing issues, please follow the provided issue template to
+ensure prompt and personalized support.
+
+### Labels
+
+Labels are used to indicate the lifecycle of all Crust firmware issues. Labels
+are prefixed with letters corresponding to the chart below.
+
+| Label/Prefix     | Description
+|------------------|------------------------------------------------------------
+| Area (A)         | Indicates the area of the area of the project impacted.
+| Difficulty (D)   | Indicates the difficulty of an issue.
+| good first issue | Indicates issues that are suitable for new contributors.
```

Comment:
Maybe put these two after the table (since they're not categories, kept to match GitHub's UI). Something like:

> Additionally, the labels "good first issue" and "help wanted" may be applied to an issue or pull request. They indicate that...